app/V1Module/router/RouterFactory.php

```diff
@@ -588,4 +590,14 @@ private static function createWorkerFilesRoutes(string $prefix): RouteList
         $router[] = new PutRoute("$prefix/result/<type>/<id>", "WorkerFiles:uploadResultsFile");
         return $router;
     }
+
+    private static function createAsyncJobsRoutes(string $prefix): RouteList
+    {
+        $router = new RouteList();
+        $router[] = new GetRoute("$prefix/<id>", "AsyncJobs:default");
+        $router[] = new GetRoute("$prefix", "AsyncJobs:list");
+        $router[] = new PostRoute("$prefix/abort/<id>", "AsyncJobs:abort");
```

Comment:
Isn't more RESTful to have `/async-jobs/<id>/abort`? and not `/async-jobs/abort/<id>`?

Reply:
True.